### DISCUSSION TITLE
Availability in package managers

### DISCUSSION THREAD

**Discussion by rgieseke** [orbitalquark/textadept#6] [Ideas]
I wonder whether it would be useful to provide more information on Textadept being available in package managers.

https://repology.org/project/textadept/versions

One could also include (selected) badges in the Readme:

https://repology.org/project/textadept/badges

This might also motivate people to update those which are out-of-date.

**Comment by orbitalquark** [comment]
I'm not sure. Some thoughts:

* Textadept releases are self-contained, so package managers may not be providing a whole lot of benefit.
* Textadept's releases are also quite frequent, so it's likely package managers will be out of date rather often.
* Some package managers (Debian I think) frown on applications that compile in their own dependencies instead of link to the ones on the system for the sake of security, so Textadept will likely never appear in those managers.
* Red badges on the README page look scary and the blame will probably fall on me more than anyone else.

**Comment by rgieseke** [comment]
Good points (especially the last one) ... I think the user maintained package managers like Homebrew are quite good at keeping up, slower ones like Debian are probably harder.

**Comment by paaguti** [comment]
Hi,
On Mon, 14 Dec 2020 at 16:50, orbitalquark <notifications@github.com> wrote:

> * Textadept's releases are also quite frequent, so it's likely package
> managers will be out of date rather often.—
>
> You are receiving this because you are subscribed to this thread.
> Reply to this email directly, view it on GitHub
> <https://github.com/orbitalquark/textadept/discussions/6#discussioncomment-205309>,
> or unsubscribe
> <https://github.com/notifications/unsubscribe-auth/AAZKU4FKVZLR5FBFMIFS33DSUYX37ANCNFSM4UZQ5L4Q>
> .
>
We have to live with that. If you want the latest and best, you better
compile it yourself :-) That's what I do, trying to follow the structure of
Textadept + separate packages for each module (I use) on  my Ubuntu.

Just mu 0.002 cents , /PA
-- 
Fragen sind nicht da um beantwortet zu werden,
Fragen sind da um gestellt zu werden
Georg Kreisler

**Comment by BrandonIrizarry** [comment]
Well, for what it's worth, I took the Slackbuild for Textadept 10.8 (the official version on Slackbuilds.org!) and updated it to 11. The only think my version lacks (I'm new to Slackware) is that it looks like I'd need to explicitly recompile the curses version to work with ncurses version 6.

**Comment by orbitalquark** [comment]
Yeah, we're at an unfortunate state where ncurses is version 5 in some platforms, but 6 on others. The binaries are built on Ubuntu 18.04, which uses ncurses 5.

**Comment by paaguti** [comment]
You may want to take a look at how I build TextAdept on Ubuntu 20.04.
https://github.com/paaguti/debian_ta
Building with Docker allows me to "back-build" for Ubuntu 1804 and for different Debian versions...
